frontend/src/metabase/query_builder/components/expressions/ExpressionEditorTextfield.jsx

```diff
@@ -62,8 +62,8 @@ const HelpText = ({ helpText, width }) =>
         <p className="text-code m0 text-body">{helpText.example}</p>
       </div>
       <div className="p2 border-top">
-        {helpText.args.map(({ name, description }) => (
-          <div>
+        {helpText.args.map(({ name, description }, index) => (
+          <div key={index}>
```

Comment:
This is good :+1:

Reply:
@ariya thanks, but that was not the conflicting diff. I am concerned about this one:
![image](https://user-images.githubusercontent.com/31325167/117175038-aea74c80-adce-11eb-9e4c-9838adad9f24.png)

The "current change" was your edit that's already on `master. Incoming change was @alxnddr 's fix for the missing key prior to your change. At least that's how I understood it. 

Just want to double-check before the merge.

Reply:
Confirmed this with @alxnddr offline. Merging...